### DISCUSSION TITLE
How to change the AstroWind Header to My Personal Header

### DISCUSSION THREAD

**Discussion by Kachi-Okorie** [arthelokyo/astrowind#296] [Q&A] [answered]
Please I want to change the AstroWind label on the header and footer to that of my website, how do I do that?
![astrowind label](https://github.com/onwidget/astrowind/assets/51110360/81f66706-7db8-43e0-92c0-0884b3ce7373)

**Comment by tambuildsthings** [comment, marked as answer]
In components/Logo.astro you have the logo.
In utils/config you have DEFAULT_SITE_NAME.
